File: L2, ling. C/operadora.c
#include <stdio.h>
double valor;
int minutos;
int main()
{
    printf("Digite a quantidade de minutos consumida: " );
    scanf("%d", &minutos);
    if(minutos<=100)
    {
        printf("O valor da conta e R$ 50,00");
    }
    else
    {
        valor=(minutos-100)*2.00+50;
        printf("O valor da conta e R$ %2.lf", valor);
    }
    return 0;
}

File: L1, ling. C#/operadora.cs
using System;
using System.Globalization;
class Program {
    static void Main() {
        int minutos;
        double valor;
        CultureInfo CI = CultureInfo.InvariantCulture;
        Console.Write("Digite a quantidade de minutos usados: ");
        minutos = int.Parse(Console.ReadLine());
        valor = 50.0;
        if (minutos > 100) {
            valor += (minutos - 100) * 2.0;
        }
        Console.WriteLine($"Valor a pagar = R$ {valor:F2}");
    }
}
